### DISCUSSION TITLE
deployment without remote storage

### DISCUSSION THREAD

**Discussion by rG2ee** [jorge-menjivar/unSAGED#14] [General]
Hello, is it possible to deploy unSAGED without a remote supabase database?

I have a problem with storing a history of my conversations in a database on a remote server.
I have a docker compose setup with a postgres database in mind.
Is that feasable somehow?

**Comment by jorge-menjivar** [comment]
At this moment it is not directly possible. There used to be support for both local and other RDBMS databases by using TypeORM. However, this was removed to make the installation more streamlined. The easiest solution now would be to update the old code that was removed and replace the supabase folder with it. Let me know if you are interested in doing this, I can re-publish that code.
